### PR TITLE
LARP Expansion Pack DLC - Tactical lying down

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Added the Ability to lie down intentionally, tactical lying down is now toggable with a middle mouse button on the walk/run icon.

Lying down tactically let you retain your weapons and items in hand, and operate them, while lying down tactically AND wielding, you get a 10% bonus on precision, recoil and fire speed, but a large movement speed penalty.

You have to unwield any weapon you might have in hand before tactically lying down or standing up from a tactical lying down.

Human and playable species mobs now change orientation, while lying down, depending on the direction they're facing, indifferently if they are lying down intentionally or not.

Added tactical lying down HUD icons, made by myself, probably not looking as nice, feel free to supply better quality icons.

Testmerge is recommended.

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!review

**Comment by SleepyGemmy** [comment]
would probably be better to just make this innate of regular resting.

File: code/_onclick/hud/screen_objects.dm [hunk 1]
```diff
@@ -460,6 +462,27 @@
...
+
+				if(C.lying_is_intentional)
+					usr.m_intent = M_WALK
+
+		if(modifiers["button"] == "middle" && !C.lying)	// See /mob/proc/update_canmove() for more logic on the lying FSM
+
+			// You want this bonus weapon or not? Wield it when you are lying, not before!
+			var/obj/item/gun/gun_in_hand = C.get_type_in_hands(/obj/item/gun)
+			if(gun_in_hand.wielded)
+				to_chat(C, "You cannot wield and lie down!")
```
Comment (NonQueueingMatt) [inline code/_onclick/hud/screen_objects.dm:482]:
```suggestion
				to_chat(C, SPAN_WARNING("You cannot wield and lie down!"))
```

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by FlamingLily** [comment]
> would probably be better to just make this innate of regular resting.

Agreed, but that being said, still add a hud icon

**Comment by feartheblackout** [comment]
Just an oversight, but this is presently unusable by Dionae because they don't have the walk/run icon.

**Comment by FluffyGhoster** [comment]
> Just an oversight, but this is presently unusable by Dionae because they don't have the walk/run icon.

Last commit should fix that, gave the hud icon back and it's prevented from running in the FSM

**Comment by listerla** [comment]
Minor thing; due to people being oriented based on direction, blankets will often cover people's heads when they're buckled to a bed.

**Comment by FluffyGhoster** [comment]
> Minor thing; due to people being oriented based on direction, blankets will often cover people's heads when they're buckled to a bed.

Fixed in commit [a7df806](https://github.com/Aurorastation/Aurora.3/pull/15952/commits/a7df8065924ef994243facdd00ccb77c4be470c0)

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**